### DISCUSSION TITLE
magical tree

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#214] [Q&A] [answered]
Another "why?"

I don't see why my solution only  get's 675% match! Ist it because of the clip-path? But I used clip-path several times with no problems. Can anyone explain me, what's the problem here? Thanks a lot!

`<style>html{margin:0 155 180;box-shadow:0 0 0 32q#998235,0 0 0 60px#1A4341,0 0 0 90px#F3AC3C,0 0 0 120px#1A4341,0 0 0 150px#998235;background:#1A4341;clip-path:inset(-150px -90px -3in -90px)}*>*{background:#F3AC3C;margin:-200 30`

**Comment by JDMeijers** [comment]
This is how it looks like for me, the clip-path must be placed on the body, not on the HTML. The background won't be visible outside of the clip-path. The clipping will cut of the edges of HTML so the background will be standard white. If you place the clipping on body, it will only clip the body and the whole HTML coloring will be visible behind it.
![Screenshot 2022-01-14 132304](https://user-images.githubusercontent.com/51381453/149514997-9999d798-d18e-412b-ae16-988327d70cd7.png)

**Comment by SelenIT** [comment, marked as answer]
As @JDMeijers said, don't rely on `clip-path` for the root element, apparently it behaves differently in the `<iframe>` (as in the CSSBattle's UI) and in a standalone "headless" browser used on the backend to check scores. You can use @alexzaworski's [previewer](https://codepen.io/alexzaworski/pen/vwmNKr) to check how the backend (most likely) will interpret your solutions.

As a side note, you probably don't need `clip-path` at all with the approach you shared. Just replace `0 0 0 150px#998235` with `0 60px 0 90px#998235` instead 😉

**Reply by verberg** [reply]
Thank's so much. Sometimes or really often I take a very big longer way... ;) but I'm learning because of this forum and all your nice helps!
